README.md

```diff
@@ -112,21 +114,32 @@ You can check that your address is funded correctly by running:
 poktrolld query bank balances [your_address]
 ```
 
-### 2. Stake your supplier
+### 2. Import your key
+While still on the node container, you should also import the key of the account you just funded so that it can be referenced in staking commands. 
+```bash
+poktrolld keys add --recover -i YOUR_KEY_NAME --keyring-dir=./localnet/poktrolld --keyring-backend test
```

Comment:
Hm, where `./localnet/poktrolld` comes from? We don't provision/use anything related to LocalNet here in this repo. We provide an example of how to deploy a production-ready stack using `docker-compose`.

Reply:
That command is run inside the container. If the README is followed through start to finish, the user may have an open terminal on the node container already from here: https://github.com/pokt-network/poktroll-docker-compose-example/pull/8/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R102